ranch/ranch.go

```diff
@@ -112,9 +112,9 @@ type acquireRequestPriorityKey struct {
 //     dest - destination state of the requested resource
 //     owner - requester of the resource
 //     requestID - request ID to get a priority in the queue
-// Out: A valid Resource object on success, or
+// Out: On success, a valid Resource object and the created time of the request, or
```

Comment:
I wonder if it'd make more sense (from an API standpoint) if this returned the duration of the request, rather than the original creation time.